app/controllers/api/policies_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::PoliciesController < Api::BaseController
+  before_action :check_permission!
+  before_action :find_resources
+
+  activate_menu :serviceadmin, :integration, :policies
+
+  sublayout 'api/service'
+
+  def update
+    if proxy.update(proxy_params)
+      redirect_to({action: :edit}, notice: 'The policies are saved successfully')
+    else
+      flash.now[:error] = 'The policies cannot be saved'
+      render :edit
+    end
+  end
+
+  protected
+
+  def proxy_params
+    params.require(:proxy).permit(:policies_config)
+  end
+
+  def check_permission!
+    provider_can_use! :api_as_product
+    authorize! :edit, service
```

Comment:
We have the service permission (which does not really work well sadly), Just thinking about the future